### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by kkiesling**
This is just some minor testing clean up for consistency and clarity.

The first update is that we use `warnings` infrastructure instead of `pytest.warns` everywhere to test warnings. This way we can check the string output of the warning (can't do that with `pytest.warns`). Some test files used `warnings` and others used `pytest` so now they are all consistent.

The second change is that we only check the string message in the warning if a warning was actually issued. In the case that there was no warning (but we expect one) then the length of the warning info is 0 and trying to do `str(w[-1].message)` gives an index error and makes the test stop. So now it's only checked if we know at least one warning was issued.

Third update is that all files now use the structure `r = np.full(X, False)` for the assertions rather than individual variables `r0`, `r1`, etc. This change is for consistency and cleanliness.

Lastly, I deleted the `not_vol.py` file. I forgot this was still in the repo. It contained all the old tests from before the refactor that I was saving for easy update of tests in the refactor. It's no longer needed.

### CODE REVIEW

**Review by gonuke — APPROVED** [review]
LGTM

File: tests/test_ivdb.py [hunk 1]
```diff
@@ -156,24 +157,27 @@ def test_make_db_dir():
...
-    with pytest.warns(None) as warn_info:
+    with warnings.catch_warnings(record=True) as w:
         iv._IvDb__make_db_dir()
-    if len(warn_info) == 1:
-        r0 = True
+        warnings.simplefilter("always")
+    if len(w) == 1:
+        r[0] = True
+        if "exists" in str(w[0].message):
+            r[1] = True
```
Comment (gonuke) [inline tests/test_ivdb.py:173]: non-blocking suggestion: There is maybe some argument for a little method that does this test?  Something like:
```
r[0], r[1] = checkWarning(w,"exists")
```

**Comment by kkiesling** [comment]
I forgot about this, but I added a method for testing warnings. Should be ready to go now.

**Comment by gonuke** [comment]
Thanks @kkiesling

**Pull request merged**